crates/optimism/node/src/args.rs

```diff
@@ -37,6 +39,14 @@ pub struct RollupArgs {
     /// Enable transaction conditional support on sequencer
     #[arg(long = "rollup.enable-tx-conditional", default_value = "false")]
     pub enable_tx_conditional: bool,
+
+    /// HTTP endpoint for the supervisor
+    #[arg(long = "rollup.supervisor-http", value_name = "SUPERVISOR_HTTP_URL")]
```

Comment:
I think you want to use `default_value = "http://localhost:1137".to_string()` here, same for safety level

Reply:
Hm, I put this into the default impl, but it looks like clap won't use it?

Reply:
exactly, for safety level use `default_value_t`. ref https://github.com/clap-rs/clap/discussions/3640